lib/IRGen/IRGenModule.cpp

```diff
@@ -367,17 +367,39 @@ IRGenModule::IRGenModule(IRGenerator &irgen,
   // TODO: Check that the deployment target supports the new
   // calling convention. Older versions of the runtime library
   // may not contain the entries using the new calling convention.
-
+  //
   // Only use the new calling conventions on platforms that support it.
+  //
+  // NB: Runtime is built with target clang, swiftc (this file) is
```

Comment:
There are only two Clangs, but there are three relevant platforms. Since there actually are "target libraries" (i.e. libraries built for the final target, rather than libraries that get linked into the compiler) I would avoid the term "target clang" in favor of "just-built clang".